docs/media-controller.md

```diff
@@ -272,3 +294,31 @@ As we work through other common use cases, both internally and with the communit
 - Example use cases:
   - [Basic example](https://media-chrome.mux.dev/examples/basic.html) ([view source](https://github.com/muxinc/media-chrome/blob/main/examples/basic.html))
   - [Spotify theme](https://media-chrome.mux.dev/examples/themes/spotify-theme.html) ([view source](https://github.com/muxinc/media-chrome/blob/main/examples/themes/spotify-theme.html))
+
+## Keyboard Shortcuts
+
+By default, Media Controller has keyboard shortcuts that will trigger behavior when specific keys are pressed when the focus is inside the Media Controller.
+The following controls are supported:
+| Key     | Behavior          |
+|---------|-------------------|
+| Space   | Toggle Playback   |
+| `k`     | Toggle Playback   |
+| `m`     | Toggle mute       |
+| `f`     | Toggle fullscreen |
+| ⬅       | Seek back 10s     |
+| ➡       | Seek forward 10s  |
+
+If you are implementing an interactive element that uses any of these keys, you can stopPropagation in your `keyup` handler. Alternatively, you can add a `keysUsed` property on the element or a `keysused` attribute.
```

Comment:
chore(blocking):

1. Discuss and link to https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values
2. Callout the `"Space"` keyword mapping

Reply:
updated